.gitignore

```diff
@@ -130,3 +128,11 @@ librz/include/sdb
 peda-session-*
 /.vs
 .cache/
+subprojects/capstone-bundled/
```

Comment:
so the reason I did not do this is that when a meson subproject is detected as outdated, meson.build suggests to use `git clean -dxff subprojects/` to clean everything and start from scratch...

However since the message is `-dxff` it should work even if these dirs are in gitignore, right?

Reply:
Yes I just checked
```
> git clean -dxff subprojects/
Removing subprojects/capstone-bundled/
Removing subprojects/packagecache/
Removing subprojects/sdb/
Removing subprojects/tree-sitter-0.18.0/
```

```
> ls subprojects
capstone-bundled.wrap  capstone-v3.wrap  libuv.wrap   lz4.wrap      sdb.wrap          xxhash  zlib.wrap
capstone-next.wrap     capstone-v4.wrap  libzip.wrap  packagefiles  tree-sitter.wrap  yxml
```